internal/manager/config/init.go

```diff
@@ -20,7 +22,28 @@ type flagStruct struct {
 	nobrowser      bool
 }
 
-var flags flagStruct
+var (
+	flags flagStruct
+
+	defaultConfigLocations = []string{
+		"config.yml",
+		filepath.Join("$HOME", ".stash", "config.yml"),
```

Comment:
This won't work on Windows

Reply:
Good catch. `$HOME` is handled by viper but it doesn't look like `koanf` does.